horizon/static/framework/widgets/table/hz-dynamic-table.html

```diff
@@ -72,7 +72,6 @@
             </div>
           </td>
           <td style="width: 4em;max-width: 5em;" ng-show="config.expand" class="expander">
-            <img title="Chameleon Supported Appliance" style="float:right;" ng-if="item.project_supported" height="20px" src="https://www.chameleoncloud.org/static/images/logo-small.png" />
```

Comment:
Do things break if you leave the icon here?

Reply:
No, but it seemed quite ugly to leave it there